app/redux/actions/async.js

```diff
@@ -764,7 +763,7 @@ export function updateSettings(api, patientId, settings) {
  * @param {userId} userId
  * @param  {Object} formValues
  */
-export function updateUser(api, formValues) {
+export function updateUser(api, formValues, options) {
```

Comment:
This new function param is never read, and no new calls seem to pass it in. Was it tacked on accidentally?
```suggestion
export function updateUser(api, formValues) {
```

Reply:
Ah yes, I'd been going about this a bit differently when I began this ticket, and accidentally left it in.